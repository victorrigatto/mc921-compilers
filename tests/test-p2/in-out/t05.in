int N = (1 * (4 + 5 - 6)) % 2 / 67;
