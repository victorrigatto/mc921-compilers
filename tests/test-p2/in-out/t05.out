Program:
    GlobalDecl:
        Decl: ID(name=N)
            VarDecl:
                Type: int @ 1:1
            BinaryOp: / @ 1:10
                BinaryOp: % @ 1:10
                    BinaryOp: * @ 1:10
                        Constant: int, 1 @ 1:10
                        BinaryOp: - @ 1:15
                            BinaryOp: + @ 1:15
                                Constant: int, 4 @ 1:15
                                Constant: int, 5 @ 1:19
                            Constant: int, 6 @ 1:23
                    Constant: int, 2 @ 1:29
                Constant: int, 67 @ 1:33
